doc/specs/vulkan/chapters/extensions.txt

```diff
@@ -223,6 +223,16 @@ wrapped by the validation layers.
 See the <<LoaderAndValidationLayers, "`Vulkan Loader Specification and
 Architecture Overview`">> document for additional information.
 
+[NOTE]
+.Note
+====
+The Specification does not currently include necessary extensions in Valid
```

Comment:
Not sure why you changed "required extensions" (defined in the glossary) to "necessary extensions" (which is not used elsewhere in the spec)..

Reply:
Mostly because I do not like to throw keywords around. "**Required** extension" to me sounds like the implementation **must** support it. You are right though it is not changed elsewhere (Glossary).